service/src/main/java/org/apache/griffin/core/job/JobController.java

```diff
@@ -113,4 +113,10 @@ public JobHealth getHealthInfo() {
                 .contentType(MediaType.APPLICATION_OCTET_STREAM)
                 .body(resource);
     }
+
+    @RequestMapping(value = "/jobs/trigger/{id}", method = RequestMethod.GET)
```

Comment:
The job trigger process is not idempotent, "GET" method seems not proper here. I think "POST" method would be better, as it creates a job submission.